error TS5083: Cannot read file 'project/.nuxt/tsconfig.json'.

### Environment

github actions


### Reproduction

- create Nuxt project with `nuxi` and `vite`
- install test framework (form me it is `jest`)
- rune `yarn test` before `yarn dev`

### Describe the bug

It seems like i can't run a test before I run the initial dev setup `yarn dev`, because `ts-config.json` extends from `.nuxt/ts-config.json`
![image](https://user-images.githubusercontent.com/60586899/141690681-d44f455a-a00e-4cea-872e-608482543956.png)


### Additional context

_No response_

### Logs

```shell
yarn run v1.22.17
$ jest
FAIL __test__/components/atom/h-switch.test.ts
  ● Test suite failed to run

    error TS5083: Cannot read file '/home/runner/work/project/.nuxt/tsconfig.json'.
    message TS151001: If you have issues related to imports, you should consider setting `esModuleInterop` to `true` in your TypeScript configuration file (usually `tsconfig.json`). See https://blogs.msdn.microsoft.com/typescript/2018/01/31/announcing-typescript-2-7/#easier-ecmascript-module-interoperability for more information.

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        1.329 s
Ran all test suites.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/packages/nuxi/src/commands/prepare.ts
+++ b/packages/nuxi/src/commands/prepare.ts
@@ -1,4 +1,5 @@
 import { resolve } from 'pathe'
+import { clearDir } from '../utils/fs'
 import { loadKit } from '../utils/kit'
 import { writeTypes } from '../utils/prepare'
 import { defineNuxtCommand } from './index'
@@ -15,6 +16,7 @@ export default defineNuxtCommand({
 
     const { loadNuxt } = await loadKit(rootDir)
     const nuxt = await loadNuxt({ rootDir })
+    await clearDir(nuxt.options.buildDir)
 
     await writeTypes(nuxt)
   }

